plugins/UsersManager/Repository/UserRepository.php

```diff
@@ -126,12 +126,14 @@ public function generateInviteToken(string $userLogin, $expiryInDays = null): st
 
     protected function sendUserCreationNotification(string $createdUserLogin): void
     {
-        $mail = StaticContainer::getContainer()->make(UserCreatedEmail::class, [
-            'login'        => Piwik::getCurrentUserLogin(),
-            'emailAddress' => Piwik::getCurrentUserEmail(),
-            'userLogin'    => $createdUserLogin,
-        ]);
-        $mail->safeSend();
+        if (Piwik::getCurrentUserLogin() !== 'anonymous') {
```

Comment:
Just a thought — should the responsibility be on this method or on the places where it's called from?